Add status alert page

emersonlebleu opened the pull request:
Added an alternative page to show a status alert when the status of the application is not up to date and in working order.

Config is customized to display maintenance for Genepanel. `/status-alert` Folder is in the root as that seemed to be the standard for this repo. It contains the js, css, and alternative index.html to display the status page.

emersonlebleu committed:
Add status alert page

Added an alternative page to show a status alert when the status of the application is not up to date and in working order.

Customized for maintenance for Genepanel.

emersonlebleu committed:
Remove complex status add simple

anderspitman commented on status_page.html:
```diff
@@ -0,0 +1,97 @@
+        
+        <div class="details">
+            <div class="detail-item">
+                <span class="detail-label">Check back in on this page in a few weeks, or contact us with quesitons.</span>
```
Typo "quesitons"

anderspitman commented on status_page.html:
```diff
@@ -0,0 +1,97 @@
+    <div class="container maintenance">
+        <div class="icon">🔧</div>
+        
+        <h1>Genepanel.iobio</h1>
```
Can you lowercase "genepanel.iobio"?

anderspitman commented on status_page.html:
```diff
@@ -0,0 +1,97 @@
+        <h1>Genepanel.iobio</h1>
+        
+        <p class="message" id="statusMessage">
+            This page is currently under maintenance to improve your experience.
```
Let's say "this app" instead of "this page"

emersonlebleu committed:
Resolve Anders' Comments

Changed genepanel to lower case in header, change reference of "page" to "app" spelled questions correctly
